package.json

```diff
@@ -3,19 +3,33 @@
   "private": true,
   "author": "Nick Barry",
   "license": "MIT",
+  "scripts": {
+    "compile": "hardhat compile && yarn copyAbi && yarn typechain",
```

Comment:
Prefer to override the compile task so that the native `hh compile` functions as expected.

Reply:
Will be taken care with the plugin https://www.npmjs.com/package/hardhat-typechain